sdk/appcenter-distribute/src/main/java/com/microsoft/appcenter/distribute/download/manager/DownloadManagerRequestTask.java

```diff
@@ -45,11 +45,16 @@ protected Void doInBackground(Void... params) {
             request.setVisibleInDownloadsUi(false);
         }
         long enqueueTime = System.currentTimeMillis();
-        long downloadId = downloadManager.enqueue(request);
-        if (isCancelled()) {
-            return null;
+        try {
+            long downloadId = downloadManager.enqueue(request);
+            if (!isCancelled()) {
+                mDownloader.onDownloadStarted(downloadId, enqueueTime);
+            }
+        } catch (IllegalArgumentException e) {
+            // In cases when Download Manager application is disabled, IllegalArgumentException: Unknown URL content://downloads/my_download is thrown.
+            AppCenterLog.error(LOG_TAG, "Failed to start download: Download Manager is disabled.");
```

Comment:
`onDownloadError` is a good point, but if we're going to use it - no need to duplicate a log